src/NovelRT/Graphics/GraphicsResourceManager.cpp

```diff
@@ -86,86 +111,138 @@ namespace NovelRT::Graphics
         stagingBuffer->UnmapAndWrite(vertexRegion);
         currentContext->Copy(vertexBuffer, stagingBuffer);
 
-        currentContext->EndFrame();
-        _graphicsDevice->Signal(currentContext->GetFence());
-        _graphicsDevice->WaitForIdle();
-        currentContext->BeginFrame();
```

Comment:
Are these just called somewhere else internally now? Or do we not need these calls anymore?

Reply:
> What is the new behavior (if this is a feature change)?
The GraphicsResourceManager will now just append to the current cmd list, so frames may consist of both resource loading and actual drawing + presenting.

It is assumed you are in a frame when you call these methods as per the behavioural change outlined in the ticket